src/tests/office_supplies/TestInkToner.java

```diff
@@ -0,0 +1,50 @@
+package office_supplies;
+
+import common.Common;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.WebDriver;
+import pages.office_supplies.InkTonerPage;
+import pages.office_supplies.OfficeSupplies;
+
+/**
+ * Created by ddantas on 4/3/2017.
+ */
+public class TestInkToner {
+
+   protected WebDriver driver;
+
+
+
+    @Before
+    public void setUp() throws Exception {
+        Common cm = new Common();
+
+        driver = cm.getDriver();
+    }
+
+    @Test
+    public void verifyHeaderText() throws Exception {
+
+        InkTonerPage inkTonerPage = new InkTonerPage(driver);
+
+        inkTonerPage = inkTonerPage.navigateToInkTonerpage();
```

Comment:
What is different between line 32 and 34? if Line 32 can take you to InkToner Page then why you need to call navigateToInkTonerpage()?